stdlib/src/builtin/object.mojo

```diff
@@ -490,9 +382,26 @@ struct _ObjectImpl(
             return "list"
         if self.is_func():
             return "function"
+        if self.is_dict():
+            return "dict"
+        if self.is_tuple():
+            return "tuple"
         debug_assert(self.is_obj(), "expected a generic object")
         return "obj"
 
+    def ref_count(ref [_]self) -> Int:
+        if self.is_dict():
+            return self.value[_RefCountedDict].impl.count().__int__()
```

Comment:
**Suggestion** Any reason not to use `int(...)` rather than calling the dunder directly?

Reply:
Nope (just a habit), fixed with next commit :+1: